proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_sv_SE.properties

```diff
@@ -0,0 +1,67 @@
+velocity.error.logging-in-too-fast=Du anslutar dig för snabbt, försök igen senare.
+# Perhaps replace "klient" with "spel"? I feel klient is a little too technical/unusual in normal speech.
+# Kanske vi borde ersätta "klient" med "spel"? Det känns som klient är lite för tekniskt/onaturligt i vardagligt tal.
+velocity.error.online-mode-only=Du är inte inloggad på ditt Minecraft konto. Om du är inloggad på ditt konto, försök att start om din Minecraft klient.
+velocity.error.player-connection-error=Ett fel med din anslutning inträffade.
+velocity.error.modern-forwarding-needs-new-client=Den här servern stöder endast Minecraft 1.13 och nyare.
+velocity.error.modern-forwarding-failed=Din server skickade inte någon vidarebefordran till proxyservern. Se till att servern är konfigurerad för Velocitybefordran.
+velocity.error.moved-to-new-server=Du blev sparkat ut ur {0}: {1}
+velocity.error.no-available-servers=Det finns inte någon tillgängliga servrar att ansluta dig till. Försök igen senare eller kontakta en administratör.
+
+# Commands
+velocity.command.generic-error=Ett fel med kommandot inträffade.
+velocity.command.command-does-not-exist=Detta kommandot finns inte.
+
+velocity.command.players-only=Endast spelare får köra det här kommandot.
+velocity.command.server-does-not-exist=The specified server {0} does not exist.
+# Perhaps/kanske: Servern {0} finns inte.
+velocity.command.server-does-not-exist=Den angivna servern {0} finns inte.
+
+velocity.command.server-current-server=Du är för tilfället ansluten till {0}.
+velocity.command.server-too-many=För många servrar är sätts upp. Använd tabslutförande för att se alla tillgängliga servrar.
+velocity.command.server-available=Tillgängliga servrar:
+velocity.command.server-tooltip-player-online={0} spelare ansluten
+velocity.command.server-tooltip-players-online={0} spelare ansluten
+velocity.command.server-tooltip-current-server=För tilfället ansluten till denna servern
+velocity.command.server-tooltip-offer-connect-server=Klicka här för att ansluta till denna servern
+
+velocity.command.glist-player-singular={0} spelare är för tilfället ansluten denna proxyservern.
+velocity.command.glist-player-plural={0} spelare är för tilfället ansluten denna proxyservern.
+velocity.command.glist-view-all=För att se alla anslutna spelare till servrar, använd /glist all.
+
+velocity.command.reload-success=Velocity konfigurationen laddades om.
+# Information? Detaljer?
+velocity.command.reload-failure=Kan inte ladda om din Velocitykonfiguration. Kolla konsolen för mer detaljer.
+
+velocity.command.version-copyright=Upphovsrätt 2018-2021 {0}. {1} är licensierat enligt villkoren i GNU General Public License v3.
+
+velocity.command.no-plugins=Det är inte någon installerade plugins.
+velocity.command.plugins-list=Plugins: {0}
+velocity.command.plugin-tooltip-website=Webbsajt: {0}
+velocity.command.plugin-tooltip-author=Auktor: {0}
+velocity.command.plugin-tooltip-authors=Auktorer: {0}
```

Comment:
"Autkor" is never really used, something like "Skapare" would probably be better